ui/pom.xml

```diff
@@ -17,9 +17,8 @@
   <name>Debezium UI Frontend</name>
 
   <properties>
-    <!-- UI Build Tools -->
-    <node.version>10.20.1</node.version>
-    <yarn.version>1.22.5</yarn.version>
+    <node.version>14.16.0</node.version>
```

Comment:
To be discussed. UI build and UI web app works without issues (local testing). Node 10 EOL is end of April. So we should update soon. Node 14 is LTS since last October. General recommendation was to wait until Node 14 became LTS, which is now for over half a year.

Reply:
I did a local test build and tested the UI with node 14 as it is here.  I think we are ok to upgrade unless @uidoyen or @indraraj see an issue with it